sentences/pl/_common.yaml

```diff
@@ -46,11 +46,17 @@ lists:
         out: "fahrenheit"
 expansion_rules:
   name: "{name}"
+  set: "(ustaw | zmień | przestaw)"
   area: "[w|na] {area}"
   what_is: "Jak[a|i] jest"
+  all: "(wszystkie | całe)"
   how_many_is: "Ile jest"
-  turn_on: "(włącz | zapal | oświeć | uruchom )"
-  turn_off: "(wyłącz | zgaś | zatrzymaj )"
+  turn_on: "(włącz | zapal | oświeć | uruchom)"
+  turn_off: "(wyłącz | zgaś | zatrzymaj)"
+  open: "(otwórz)"
+  open_blind: "(odsłoń | podnieś)"
```

Comment:
Just wondering, should we also add `otwórz` to open_blinds, or will it work?
User may say `otwórz rolety w sypialni` (open blinds in the bedroom) or `zamknij rolety w symialni` (close blinds in the bedroom).

Reply:
Yes `otwórz` should and in fact is a part of blinds opening extensions 😄 
This is why @witold-gren used the `"(<open> | <open_blind>) rolety <area>"` syntax. I really like this approach because it gives you full flexibility by both covering the sentences like in your example while also keeping a clear separation between general extensions and these for blinds.

Reply:
@Uriziel01 thank you for a quick reply. I need to read how to test the sentences, this will probably answer many questions before posting them here.
What about this part - https://github.com/home-assistant/intents/pull/534/files#diff-8cdabf72346a4b01ae6dafb0a8b29b8d72b1c5078c2726219659fcf57d61043dR15
`"(<open> | <open_blind>) rolety <area>"`
We should be able to say/type `Otwórz rolety salon`, but what about `Otwórz rolety w salonie`?
We will be able to add aliases to entities and areas in HA, but what about `w`?

Reply:
No problem. The documentation mentioned in readme file in really good, you can use that to learn the syntax and how to test it properly.
And to answer your question - this one is handled by the area extension here `area: "[w|na] {area}"` so it covers cases like `w kuchni` or `na balkonie`. Of course after you declare all the forms for an area in https://github.com/home-assistant/intents/blob/main/tests/pl/_fixtures.yaml

Reply:
I've run some tests, I've added two covers to living room (in _fixtures), and I have another set of questions:
`otwórz rolety w salonie` works fine, but `otwórz wszystkie rolety w salonie` doesn't return match.
Same with `otwórz wszystkie rolety`.
Maybe let's discuss this here: https://github.com/home-assistant/intents/discussions/388

Reply:
This is because this sentence https://github.com/home-assistant/intents/pull/534/files#diff-8cdabf72346a4b01ae6dafb0a8b29b8d72b1c5078c2726219659fcf57d61043dR16 uses another word ordering. And yes, I agree that putting `wszystkie` before the area extensions would be way more natural way of saying it.

Reply:
so we should add another option, like this:
`"(<open> | <open_blind>) [wszystkie] rolety <area>"`
I think a similar approach should be used with other platforms, you can turn on a single light source or all lights in a specific room (area) or even in entire house

Reply:
It is added in some places, not all though. To not get @witold-gren PR even more complicated than it is currently I suggest to only fix ordering here to a correct one and leave the rest of cases for now.

I want to avoid having hanging PRs for prolonged amount of time because we figure out new ways to make it better.

Reply:
hey, I also fixed problem with `(<open> | <open_blind>) [wszystkie] rolety <area>`

Reply:
OK, so I also added support for "all devices" without set area. So now we can say `Zamknij wszystkie rolety` , `Wyłącz wszystkie wentylatory` or `Zgaś wszystkie światła` 😉

Reply:
My proposition is just close this PR and not add more rules, because now it is quite big. If we need add more functionality we should create another PR or list of requested feature..

Reply:
It's already merged 48mintues ago @witold-gren 😉